packages/utils/src/string/isValidEmail/index.ts

```diff
@@ -0,0 +1,5 @@
+export const isValidEmail = (email: string) => {
+  const regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
+
+  regex.test(email) ? true : false;
```

Comment:
return문이 빠져있습니다. return문을 추가해주시면 감사드립니다.

```suggestion
  return regex.test(email);
```

Reply:
수정했습니다. 확인 부탁드립니다